examples/scripts/osft_mlflow_example.py

```
@@ -0,0 +1,139 @@
+
+
+def main():
+    parser = argparse.ArgumentParser(description='OSFT Training with MLflow Logging')
+
+    # Model and data paths
+    parser.add_argument('--model-path', default='Qwen/Qwen2.5-7B-Instruct',
+                       help='Model path or HuggingFace name (default: Qwen/Qwen2.5-7B-Instruct)')
+    parser.add_argument('--data-path',
+                       required=True,
+                       help='Path to training data (JSONL format)')
+    parser.add_argument('--ckpt-output-dir', default='/tmp/osft-mlflow-checkpoints',
```

Ruff (0.14.14)

Probable insecure usage of temporary file or directory: "/tmp/osft-mlflow-checkpoints"

(S108)